src/internal/nativeModule.web.ts

```diff
@@ -75,7 +75,7 @@ declare global {
 const isWindowPresent = typeof window !== 'undefined';
```

Comment:
I wonder about this comment

```
// Use a constant test of this form because in SSR on next.js, optional chaining is not sufficient,
// but this test correctly detects that window is not available and allows for conditionals before access
```

and if your additional conditionals should be added up in the `const isWindowPresent` line? I don't deploy on next.js so can't test it but don't want to break them